source/includes/extracts-atlas-search-facts.yaml

```
@@ -0,0 +1,27 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  **For data hosted on** :atlas:`MongoDB Atlas </>`, MongoDB offers a 
```

[suggestion]

This was a little visually jarring when I first loaded the page. I think we'd be better off dropping the link to the Atlas docs and bolding the entire "For data hosted on MongoDB Atlas".

```suggestion
  **For data hosted on MongoDB Atlas**, MongoDB offers a 
```